src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -197,21 +197,41 @@ cdef class _ServicerContext:
     def set_trailing_metadata(self, object metadata):
```

Comment:
Please also update the interface class to include the new methods: https://github.com/grpc/grpc/blob/master/src/python/grpcio/grpc/aio/_base_server.py#L134. Since they will be new APIs, please add "This is an EXPERIMENTAL API." in docstrings. We don't need to add "@abc.abstractmethod" to the new methods, since we don't want to break existing implementations of the `ServicerContext` interface.